WebRoot/css/zm.css

```diff
@@ -3572,22 +3572,60 @@ HTML>BODY .appt-selected .appt_allday_body
 }
 
 .ZmContactSearch {
-	width:500px;
+	@ContactSearchWidget@
 }
+
 .ZmContactSearch .contactSearchResultsDiv {
-	margin-top:15px;
+	@ContactSearchResultsContainer@
+}
+
+.ZmContactSearch .DwtListView-ColHeader {
+	@ContactSearchListColHeader@
```

Comment:
Using ZmContactSearchListView.prototype.createHeaderHtml to return an empty list, now that overrides the listview's createHeaderHtml .